business_requirement_sale/wizard/business_requirement_create_sale.py

```diff
@@ -92,35 +92,30 @@ def _onchange_undefined_section(self):
         command = 4 if self.undefined_section else 3
         self.deliverable_ids = [(command, x) for x in deliverables.ids]
 
-    def _prepare_sale_layout_category_vals(self, deliverable, line_vals):
-        return {
-            'name': deliverable.section_id.name,
-        }
-
-    def _prepare_sale_order_line_vals(self, deliverable, order_vals):
+    def _prepare_sale_order_line_vals(
+            self, deliverable, order_vals, is_section=False):
```

Comment:
Don't use the same method for sections than for sales order lines. Put a different one. You must make sure also through a sequence than the section is before the lines it contains, and also that all lines with same section are together.